### ISSUE TITLE
Allow regular users to leave signature reviews (TSig_Review)

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/BugSigDB#203]
If not difficult, I think it would be better to allow regular users to leave reviews on the taxa in signatures without requiring "Reviewer" permissions (ie the green speech bubbles below):

<img width="104" alt="image" src="https://github.com/waldronlab/BugSigDB/assets/639942/adb665b7-023a-472f-b7ed-8a3aa0b49034">

**Comment by tosfos** [comment]
I think we can do that.

**Comment by tosfos** [comment]
Is it OK if regular users are also allowed to add reviews for the Study, Experiment & Signature pages? It's easier that way. We can also accommodate allowing them to only review taxa.

**Comment by lwaldron** [comment]
Yes I think that's fine.

**Comment by lwaldron** [comment]
By "add reviews" for Study, Experiment, & Signature pages, do you mean marking the page as reviewed? If any user can mark those pages as reviewed then as far as I know there will be no additional privileges for "Reviewer". That's maybe suboptimal but I don't think a big deal and I can live with it.

**Comment by tosfos** [comment]
That is correct. We'll check how difficult it will be to separate the 2 different types of reviews and we'll fall back to allowing all reviews if needed.

**Comment by tosfos** [comment]
We added a solution to production and it is undergoing testing. Will update soon.

**Comment by tosfos** [comment]
Regular users should now be able to leave reviews on the taxa in signatures even if they are not in the reviewer group.

**Comment by tosfos** [comment]
Can this be closed?
